proto/query.proto

```diff
@@ -562,6 +562,36 @@ message BeginExecuteBatchResponse {
   int64 transaction_id = 3;
 }
 
+// MessageStreamRequest is the request payload for MessageStream.
+message MessageStreamRequest {
+  vtrpc.CallerID effective_caller_id = 1;
+  VTGateCallerID immediate_caller_id = 2;
+  Target target = 3;
+  // name is the message table name.
+  string name = 4;
+}
+
+// MessageStreamResponse is a response for MessageStream.
+message MessageStreamResponse {
+  QueryResult result = 1;
+}
+
+// MessageAckRequest is the request payload for MessageAck.
+message MessageAckRequest {
+  vtrpc.CallerID effective_caller_id = 1;
+  VTGateCallerID immediate_caller_id = 2;
+  Target target = 3;
+  // name is the message table name.
+  string name = 4;
+  repeated Value ids = 5;
```

Comment:
Need a comment of what ids should be in there, same as the other is fine:
// ids is the list of ids to ack.